### PR TITLE
Use Pekko name

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

**Review by pjfanning — APPROVED** [review]
lgtm

File: build.sbt [hunk 1]
```diff
@@ -286,42 +277,48 @@ lazy val `integration-test-local` = project
...
-      "extref.akka-http.base_url" -> s"https://doc.akka.io/docs/akka-http/${Dependencies.AkkaHttpBinaryVersion}/%s",
-      "scaladoc.akka.http.base_url" -> s"https://doc.akka.io/api/akka-http/${Dependencies.AkkaHttpBinaryVersion}/",
-      "extref.akka-grpc.base_url" -> s"https://doc.akka.io/docs/akka-grpc/current/%s",
-      "extref.akka-enhancements.base_url" -> s"https://doc.akka.io/docs/akka-enhancements/current/%s",
+      "extref.akka.base_url" -> s"https://https://pekko.apache.org/docs/pekko/current/%s",
+      "scaladoc.akka.base_url" -> s"https://https://pekko.apache.org/api/pekko/current/",
+      "extref.akka-http.base_url" -> s"https://https://pekko.apache.org/docs/pekko-http/${Dependencies.AkkaHttpBinaryVersion}/%s",
+      "scaladoc.akka.http.base_url" -> s"https://https://pekko.apache.org/api/pekko-http/${Dependencies.AkkaHttpBinaryVersion}/",
+      "extref.akka-grpc.base_url" -> s"https://https://pekko.apache.org/docs/pekko-grpc/current/%s",
+      "extref.akka-enhancements.base_url" -> s"https://https://pekko.apache.org/docs/pekko-enhancements/current/%s",
```
Comment (pjfanning) [inline build.sbt:311]: the https:// is repeated twice in the last 5 lines

Comment (nvollmar) [reply build.sbt:311]: thanks - fixed

**Comment by nvollmar** [comment]
@pjfanning I did quite some rework since your first review but should be ready to merge now.

File: project/Common.scala [hunk 2]
```diff
@@ -15,20 +15,21 @@ object Common extends AutoPlugin {
 
   override lazy val projectSettings: Seq[sbt.Def.Setting[_]] =
     Seq(
-      organization := "com.lightbend.akka.management",
-      organizationName := "Lightbend Inc.",
-      organizationHomepage := Some(url("https://www.lightbend.com/")),
+      organization := "org.apache.pekko.management",
+      organizationName := "Apache Software Foundation",
+      organizationHomepage := Some(url("https://pekko.apache.org/")),
```
Comment (pjfanning) [inline project/Common.scala:20]: organizationHomepage should be www.apache.org while homepage is pekko.apache.org

File: project/Common.scala [hunk 3]
```diff
@@ -15,20 +15,21 @@ object Common extends AutoPlugin {
 
   override lazy val projectSettings: Seq[sbt.Def.Setting[_]] =
     Seq(
-      organization := "com.lightbend.akka.management",
-      organizationName := "Lightbend Inc.",
-      organizationHomepage := Some(url("https://www.lightbend.com/")),
+      organization := "org.apache.pekko.management",
```
Comment (pjfanning) [inline project/Common.scala:18]: Can we use `org.apache.pekko` as organization? I think this affects the Maven coordinates and so far, we have kept everything in `org.apache.pekko` Maven groupId.

File: project/Common.scala [hunk 4]
```diff
@@ -15,20 +15,21 @@ object Common extends AutoPlugin {
...
+      organizationName := "Apache Software Foundation",
+      organizationHomepage := Some(url("https://pekko.apache.org/")),
       startYear := Some(2017),
```
Comment (pjfanning) [inline project/Common.scala:21]: startYear - maybe 2022 - 2017 is an Akka date

File: project/Common.scala (hunk 3 above)
Comment (nvollmar) [reply project/Common.scala:18]: I followed akka-management, but I don't see a reason not to use `org.apache.pekko` either.

File: build.sbt [hunk 5]
```diff
@@ -4,167 +4,178 @@ import sbt.Keys.parallelExecution
...
-lazy val `akka-discovery-kubernetes-api` = project
+lazy val discoveryKubernetesApi = project
   .in(file("discovery-kubernetes-api"))
   .enablePlugins(AutomateHeaderPlugin)
+  .settings(MetaInfLicenseNoticeCopy.settings)
   .settings(
-    name := "akka-discovery-kubernetes-api",
-    organization := "com.lightbend.akka.discovery",
+    name := "pekko-discovery-kubernetes-api",
+    organization := "org.apache.pekko.discovery",
```
Comment (pjfanning) [inline build.sbt:47]: I think we should use `org.apache.pekko` as org everywhere in this file

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**